code/game/objects/items/storage/belt.dm

```diff
@@ -27,6 +27,48 @@
 	. = ..()
 	update_icon()
 
+/obj/item/storage/belt/durathread
+	name = "durathread toolbelt"
+	desc = "A toolbelt made out of durathread, it seems resistant enough to hold even big tools like an RCD, it also has higher capacity."

```

Comment:
I feel like that last comma should be replaced by a semicolon or a period.

Reply:
Alternative phrasing:
`"A toolbelt made out of durathread. It seems resistant enough to hold larger tools, such as an RCD; it also has a higher capacity."`